src/app/asset-upload/asset-upload.component.ts

```diff
@@ -8,12 +8,39 @@ import {AssetUploadService} from './asset-upload.service';
 })
 export class AssetUploadComponent {
   file: File | null = null;
+  assetLink = '';
+  isSizeInvalid = false;
 
   constructor(private service: AssetUploadService) {}
 
+  assetLinkUpload(assetLink: string) {
```

Comment:
Can we possibly maintain the asset as an external link instead of creating a local asset blob? i.e. the final output will source to the given URL instead of a locally downloaded asset.

Reply:
Completed this in final touches PR